benchmarks/devel/devel.cpp

```diff
@@ -0,0 +1,64 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Kokkos_Core.hpp>
+
+#include <chrono>
+
+#include <benchmark/benchmark.h>
+
+template <typename ExecutionSpace>
+void BM_benchmark(benchmark::State &state)
+{
+  ExecutionSpace exec_space;
+
+  auto const n = state.range(0);
+
+  Kokkos::View<int *, typename ExecutionSpace::memory_space> view(
+      Kokkos::view_alloc(exec_space, "Benchmark::view",
+                         Kokkos::WithoutInitializing),
+      n);
+
+  for (auto _ : state)
+  {
+    exec_space.fence();
+    auto const start = std::chrono::high_resolution_clock::now();
```

Comment:
I thought it is recommended to avoid `high_resolution_clock` these days

Reply:
While true, it is defined as the highest resolution clock. Replacing with with either `steady_clock` or `system_clock` may lose precision on some systems. So I don't think we can do that.